x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/left/components/graph_visualization.tsx

```diff
@@ -42,6 +46,35 @@ export const GraphVisualization: React.FC = memo(() => {
   });
 
   const originEventIds = eventIds.map((id) => ({ id, isAlert }));
+  const { openTimeline } = useTimelineApi();
+  const openTimelineCallback = useCallback(
+    (filters: Filter[], timeRange: TimeRange) => {
+      const from = dateMath.parse(timeRange.from);
+      const to = dateMath.parse(timeRange.to);
+
+      if (!from || !to) {
+        // TODO: show error message
```

Comment:
I'm not sure how to reproduce this issue yet

Reply:
perhaps have a local error state?
```suggestion
        const errors: string[] = []
        if(!from) {
            errors.push('From error message)
        }
        if(!to) {
            errors.push('To error message)
        }

        setErrors(errors)
```